test/org/sagebionetworks/bridge/services/AuthenticationServiceTest.java

```diff
@@ -90,54 +96,76 @@
     
     @Resource
     private SubpopulationService subpopService;
-
-    private TestUser testUser;
     
-    @Before
-    public void before() {
-        testUser = helper.getBuilder(AuthenticationServiceTest.class).build();
-    }
-
-    @After
-    public void after() {
-        helper.deleteUser(testUser);
+    private TestUser makeTestUser() {
+        return helper.getBuilder(AuthenticationServiceTest.class).build();
     }
 
     @Test(expected = BridgeServiceException.class)
     public void signInNoEmail() throws Exception {
-        authService.signIn(testUser.getStudy(), TEST_CONTEXT, new SignIn(null, "bar"));
+        TestUser testUser = makeTestUser();
+        try {
+            authService.signIn(testUser.getStudy(), TEST_CONTEXT, new SignIn(null, "bar"));    
+        } finally {
+            helper.deleteUser(testUser);
+        }
```

Comment:
I think the diff is confusing. I took the creation of a test user out of a @Before method because it wasn't used in every method (and sometimes a second user was getting created), instead manually creating it in each test where it was needed and getting rid of the duplicate user in a couple of tests.